Servers/UI/OJS.Servers.Ui.Models/Submissions/Details/SubmissionDetailsResponseModel.cs

```diff
@@ -29,6 +29,8 @@ public class SubmissionDetailsResponseModel : IMapFrom<SubmissionDetailsServiceM
 
         public double MaxUsedMemory { get; set; }
 
+        public int MemoryLimit { get; set; }
```

Comment:
This is not needed. Let's remove it from submission details page for now.

Reply:
done